src/index.html

```diff
@@ -1,14 +1,28 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mo Yo App</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#" class="logo__link">
+        <img src="./images/logo.png" alt="Mo Yo Logo">
+      </a>
+      <nav>
+        <ul>
+          <li><a href="#" class="is-active">Apple</a></li>
+          <li><a href="#" >Samsung</a></li>
+          <li><a href="#" >Smartphones</a></li>
+          <li><a href="#" data-qa="hover">Laptops & computers</a></li>
```

Comment:
For readability you should consider spliting these li's (and any code that spreads for the whole screen) in multiple lines, specially when there's attributes